### DISCUSSION TITLE
Does anyone have any recommendations for scaling text sizes?

### DISCUSSION THREAD

**Discussion by lewisd1996** [nativewind/nativewind#472] [Q&A]
The difference between text sizes on my iPhone and Huawei devices are quite different, they both have regular scaling.

I don't want to limit font scaling as that harms accessibility, but I'd like to scale down my Android devices a bit to match up with iOS.

Does anyone have their config that they could share? I've played around with all the functions in [the theme section of the docs](https://www.nativewind.dev/customization/theme) but it's not worked, just bugged out for me.

**Comment by tconroy** [comment]
Also wondering this. Ever figure it out? I'd like to let users control font scaling in my app

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
